spec/models/miq_widget_spec.rb

```diff
@@ -139,7 +139,7 @@
         before do
           ws = FactoryBot.create(:miq_widget_set, :name => "Home", :owner => @user1, :userid => @user1.userid, :group => @group1)
 
-          @widget_report_vendor_and_guest_os.make_memberof(ws)
+          ws.add_member(@widget_report_vendor_and_guest_os)
```

Comment:
I like these changes as they are. And they seem common across the various MiqSet PRs
can this whole file be merged now?

Reply:
I kind of like the idea of pulling out the refactors like this first into its own PR, then they aren't in the view of the more complicated changes...it's up to you @NickLaMuro

Reply:
Done:  https://github.com/ManageIQ/manageiq/pull/21257